pyi.py

```diff
@@ -188,25 +213,46 @@ def visit_Assign(self, node: ast.Assign) -> None:
         if isinstance(assignment, ast.Call) and isinstance(assignment.func, ast.Name):
             cls_name = assignment.func.id
             if cls_name in ("TypeVar", "ParamSpec", "TypeVarTuple"):
-                typevar_name = target.id
-                if typevar_name.startswith("_"):
-                    target_info = TypeVarInfo(cls_name=cls_name, name=typevar_name)
+                if target_name.startswith("_"):
+                    target_info = TypeVarInfo(cls_name=cls_name, name=target_name)
                     self.typevarlike_defs[target_info] = node
                 else:
-                    self.error(target, Y001.format(cls_name))
+                    self.error(node, Y001.format(cls_name))
                 return
             # We allow assignment-based TypedDict creation for dicts that have
             # keys that aren't valid as identifiers.
             elif cls_name == "TypedDict":
                 return
-        if isinstance(node.value, (ast.Num, ast.Str)):
+        if isinstance(node.value, (ast.Num, ast.Str, ast.Bytes)):
```

Comment:
This seems like something that should be split into a separate PR. It seems to be also the cause of the merge conflict.

Reply:
It's because I changed some of the tests to use bytes literals so they wouldn't trigger this new check in addition to existing ones. I don't think this change will have a practical effect but it makes the tests cleaner.